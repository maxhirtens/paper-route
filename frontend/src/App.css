body {
  background: linear-gradient(#e66465, #9198e5);
  background-repeat: no-repeat;
  height: 100vh;
}

.App {
  text-align: center;
  display: flex;
  width: 100%;
  max-width: 60%;
  margin: 0 auto;
  justify-content: center;
}

.blinking {
  animation: mymove 1s infinite alternate;
  color: blueviolet;
}

@keyframes mymove {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}

