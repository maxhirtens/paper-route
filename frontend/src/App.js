import React from "react";
import RoutesList from "./routes/Routes";
import { BrowserRouter } from "react-router-dom";
import NavBar from "./navbar/NavBar";
import "./App.css";
// Bootstrap CSS
import "bootstrap/dist/css/bootstrap.min.css";
// Bootstrap Bundle JS
import "bootstrap/dist/js/bootstrap.bundle.min";

function App() {
  return (
    <BrowserRouter>
      <div className="App">
        <NavBar />
        <RoutesList />
      </div>
    </BrowserRouter>
  );
}

export default App;
