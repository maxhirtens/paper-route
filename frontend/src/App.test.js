import { render } from "@testing-library/react";
import App from "./App";

// smoke test for App component.
it("renders without crashing", function () {
  render(<App />);
});
