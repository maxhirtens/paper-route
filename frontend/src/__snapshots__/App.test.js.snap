// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer text-center text-muted"
  >
    Made with React.js by 
    <a
      href="https://github.com/maxhirtens"
    >
      @maxhirtens
    </a>
  </div>
  <div
    class="footer text-center text-muted"
  >
    Data provided by The New York Times.
  </div>
</DocumentFragment>
`;
